assets/scripts/fe/src/app/components/users/setting/setting.component.ts

```diff
@@ -27,10 +40,24 @@ export class SettingComponent implements OnInit {
 
   async ngOnInit() {
     // initialize the form.
+    this.switch_expression = null
```

Comment:
terminator missing. this causes bugs

Reply:
don't use snake case on javascript syntax.
javascript = camelCase and PascalCase
python = snake_case and PascalCase

Reply:
I also believe this flag variable is a bad idea